feature-libs/customer-ticketing/components/customer-ticketing-details/customer-ticketing-details.component.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { I18nTestingModule, TranslationService } from '@spartacus/core';
+import { CardModule } from '@spartacus/storefront';
+import { Observable, of } from 'rxjs';
+import { CustomerTicketingDetailsComponent } from './customer-ticketing-details.component';
+
+class MockTranslationService {
+  translate(): Observable<string> {
+    return of();
+  }
+}
+
+describe('CustomerTicketingDetailsComponent', () => {
+  let component: CustomerTicketingDetailsComponent;
+  let fixture: ComponentFixture<CustomerTicketingDetailsComponent>;
+
+  beforeEach(async () => {
+    TestBed.configureTestingModule({
+      imports: [I18nTestingModule, CardModule],
+      declarations: [CustomerTicketingDetailsComponent],
+      providers: [
+        { provide: TranslationService, useClass: MockTranslationService },
+      ],
+    }).compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CustomerTicketingDetailsComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
```

Comment:
I'm assuming the unit tests will be done later right?

Reply:
Yes, along with the core implementation.